WTWSMS/common/bookmarks/00_bookmarks.txt

```diff
@@ -92,6 +92,22 @@ bm_little_augustus = {
 			dynasty = 1042151
 		}
 	}
+	selectable_character = {
+		id = 30 # Aiwareiks
+		age = 42
+		name = ERA_CHAR_NAME_30
+		title = k_visigoths
+		title_name = k_visigoths
+		
+		character = {
+			dna="spmzekpekit" # From pre-2.5 save
+			properties="ak000b000000"
+			religion = arian
+			culture = visigothic
+			government = feudal_government
```

Comment:
Should presumably be Sub-Roman not Feudal?

Reply:
Fixed for both bookmarks. There are probably more selectable characters that need a government update.